doc/tutorials/basic/panes.md

```diff
@@ -0,0 +1,421 @@
+# Display Content with Panes
+
+In this tutorial, we will learn to display objects with *Panes*:
+
+- *Panes* are available in the `pn.pane` namespace.
+- *Panes* take an `object` argument as well as other arguments.
+- Discover all *Panes* and their *reference guides* in the [Panes Section](https://panel.holoviz.org/reference/index.html#panes) of the [Component Gallery](../../reference/index.md).
+
+:::{note}
+A *Pane* is a component that can display an object. It takes an `object` as an argument.
+
+A *Pane* is defined as a component that inherits from the `PaneBase` class.
```

Comment:
I put it in because I was very confused about the concepts of Panes, Widgets and Layouts in the beginning. I lacked clear definitions. And the explanations I could find from Philipp always assumed some definition that he had in his head but was not documented. For me it would have been very helpful to have this definition.

We also have the problem that we cannot make custom *Panes*, *Widgets* and *Layouts* using `ReactiveHMTL` or `Viewer` in the *they inherit from this class* sense. So defining them in this *hard* sense will only gives us troubles explaining things.

It should probably go to an explanation section. I've removed it.